### ISSUE TITLE
Duplicates in UK Master List

### ISSUE THREAD

**Issue by JimBacon** [Indicia-Team/warehouse#83]
The UK Master List contains some duplicate entries, i.e. the same taxon_id is referred to multiple times in the one list. This seems wrong to me. The following query lists such entries on warehouse1.

```
select taxon_id, count(*) from
taxa_taxon_lists ttl 
where taxon_list_id = 15
and deleted = false
group by taxon_id
having count(*) > 1
order by count(*)
desc
```

**Comment by johnvanbreda** [comment]
Agreed, I can't see a good reason for this. It's possibly a bug in the UKSI import script, which is pretty complex so hardly surprising.
